system-addon/content-src/styles/_theme.scss

```diff
@@ -34,7 +34,7 @@
   // Search
   --newtab-search-background-color: $white;
   --newtab-search-border-color: transparent;
-  --newtab-search-focus-border-color: $blue-50;
+  --newtab-search-focus-border-color: $blue-60;
```

Comment:
we could potentially take away this variable now 😈

Reply:
And just use `--newtab-button-primary-color`? There seems to be an unused `$input-primary` now that can be removed too. Let's remove both and… leave the "button" name? Unless you want to rename them to `--newtab-input-primary-color` (and button-secondary too)?

Reply:
I was thinking of just setting it to `$blue-60` directly since both themes have the same color now. Good?